lib/kmer_filters.cc

```diff
@@ -119,6 +119,22 @@ KmerFilter get_simple_label_intersect_filter(const LabelSet& src_labels,
 }
 
 
+KmerFilter get_junction_count_filter(const Kmer& src_node,
+                                     CountingHash * junctions,
+                                     const unsigned int min_cov)
+{
+    KmerFilter filter = [=] (const Kmer& dst_node) {
```

Comment:
That's a lambda function capturing by-copy (`[=]`) any variable used inside the function body. And `KmerFilter` is a type for a function that takes a `Kmer` and returns `True` or `False` (defined at https://github.com/dib-lab/khmer/blob/46572454f123d36e6744712820945cf60ba5f086/lib/khmer.hh#L166): `typedef std::function<bool (const Kmer&)> KmerFilter;`